src/ResourceManager/StorageSync/Commands.StorageSync/Commands.StorageSync.csproj.user

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@afedyashov please remove this file

Reply:
Contains configuration settings of our cmdlet. Currently they are embedded into the module (hardcoded). This might change in the future.

Reply:
Debugging is strictly for testing your cmdlets in a PowerShell session outside of tests -- the `.user` file shouldn't be checked-in (and should've been caught by the [`.gitignore`](https://github.com/Azure/azure-powershell/blob/preview/.gitignore#L25) file)

Reply:
I wonder how strongly you feel about this. Without that, any other person who works on the project, would have to repeat the steps in the guide and basically repeat everything I already did - recreate debug.ps1 and re-configure Visual Studio Debug settings, which is significantly less productive. It adds significant debugging/manual testing facility, what is the benefit of removing it? Can we get more opinions on this? @maddieclayton, @praries880